bbl_screen-patch/interpose.cpp

```diff
@@ -273,6 +273,17 @@ class X1PlusNativeClass : public QObject {
             close(fd);
         }
     }
+
+public:
+    int chamberledSetting = 1;
+    Q_INVOKABLE void updateChamberLED(int val) {
+        std::string valueText = std::to_string(chamberledSetting);
```

Comment:
This does not actually set `chamberledSetting`!!

Reply:
you're right, `chamberledSetting` was set incorrectly in a now deleted line. I'm not going to fix this one, because this PR does not need to exist

Reply:
you're right, `chamberledSetting` was set incorrectly in a now deleted line. I'm not going to fix this one, because this PR does not need to exist